src/unity/toolkits/drawing_recognition/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+project(unity_toolkits)
+
+make_library(unity_drawing_recognition
```

Comment:
Note that you'll need to update to the new CMake structure that Zach merged, using OBJECT libraries instead.

Reply:
Done